### DISCUSSION TITLE
Daemon does not work properly, only manually working.

### DISCUSSION THREAD

**Discussion by SosseHD** [DNSCrypt/dnscrypt-proxy#2273] [Q&A]
So, I installed everything and generelly it seems to work. When I type
`./dnscrypt-proxy` (im in root) it starts, and I can browse the web as normal.

But when I type `./dnscrypt-proxy -service start` it doesnt work. Sometimes the message that it started takes some time sometime it doesnt take time but thats not the problem, the problem is that when i THEN try to browse the web SOME sites work, most dont. And those that DO work, may be not working next time I `-service start` it, but then some other random sites seem to work, not to forget that when some sites work they are extremely slow in loading or its kindoff broken and not really usable. 
Like for example, text breaking, where something should be positioned and so on.

I did everything as the isntallation told me to, otherwise it wouldnt work when i simply type `./dnscrypt-proxy`
I got told on reddit that the problem may be with my OS or system config, but he didnt tell me anything further. I use Fedora 37, installed it a week ago. My first linux distro. Is there any other information I should include?

**Comment by jedisct1** [comment]
It sounds like you are running into some issues when trying to use dnscrypt-proxy in service mode on your Fedora 37 system.
It could be an issue with your OS or system configuration. Some things you can try to troubleshoot the problem include checking your firewall settings to make sure they're allowing dnscrypt-proxy traffic, and checking your dnscrypt-proxy configuration file to ensure that it's set up correctly. Additionally, you may want to check the dnscrypt-proxy log files to see if they contain any error messages that could help identify the problem.
If you're still having trouble, it may be helpful to include more information about your system configuration and any error messages you're seeing when you try to start the service.
